python_modules/dagster-pipes/dagster_pipes/__init__.py

```diff
@@ -319,13 +331,29 @@ def _param_from_env_var(key: str) -> Any:
 
 
 def encode_env_var(value: Any) -> str:
+    """Encode a value as a JSON-serialized, base64-encoded, zlib-compressed string.
```

Comment:
`base64_encode(compress(to_json(value)))`

is the invocation, so the english sentence should reflect that. The function notation itself is quite clear, so maybe that is best.

"Encode value by serializing to JSON, then compressing with zlib, and then finally encoding with base64"
`base64_encode(compress(to_json(value)))` in function notation"